plugins/arDominionPlugin/css/less/jobs.less

```diff
@@ -48,7 +48,7 @@
   margin: 5px 5px 0px 5px;
   padding: 5px;
   background-color: #f2f2f2;
-  border-radius: 10px;
+  border-radius: 4px;
   color: @black;
   font-family: Courier New;
   max-height: 400px;
```

Comment:
Yes, the job log might be huge in the future. We'll be moving all the CSV / XML import tasks to job scheduler soon, the output of these tasks might be hundreds of lines if it's a large import.